tests/suites/0_stateless/09_fuse_engine/09_0014_func_clustering_information_function_cluster.result

```diff
@@ -0,0 +1,6 @@
+1	1
+2	1
+0	3
+1	3
+4	4
+ERROR 1105 (HY000) at line 9: Code: 1002, displayText = Unimplement error.
```

Comment:
This error message is not stable with the `at line 9:`.
Can we set an ErrCode number for it?  @zhang2014 @sundy-li

Reply:
Why not use ` -- {ErrorCode 1002}`

Reply:
The `-- {ErrorCode 1002}` should be set on the `09_0014_func_clustering_information_function.sql`, but this test should not work with standalone.

Reply:
Let me comment out the case first, this bug will be fixed soon.